painttiming.bs

```diff
@@ -80,9 +82,7 @@ The {{PerformancePaintTiming}} interface {#sec-PerformancePaintTiming}
 * The {{PerformanceEntry/startTime}} attribute must return a {{DOMHighResTimeStamp}} of when the paint occured.
 * The {{PerformanceEntry/duration}} attribute must return 0.
 
-A user agent implementing {{PerformancePaintTiming}} must perform the following steps:
-1. Run the <a>register a performance entry type</a> algorithm with <code>"first-paint"</code> as input.
-1. Run the <a>register a performance entry type</a> algorithm with <code>"first-contentful-paint"</code> as input.
+NOTE: A user agent implementing {{PerformancePaintTiming}} is compelled to include <code>"first-paint"</code> and <code>"first-contentful-paint"</code> in {{PerformanceObserver/supportedEntryTypes}} for {{Window}} contexts.
```

Comment:
s/is compelled/would need to/ ?
Maybe also add "so that developers would be able to detect the feature's support" or some such?